### ISSUE TITLE
SpecsFinder doesn't appear to find specs that don't directly mix-in Specification trait

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#86]
```
What steps will reproduce the problem?
1. Declare a class that extends Specification, e.g. "MyParent"
2. Create a class that extends "MyParent", e.g. "MyChild". MyChild does not
explicitly say "with Specification", it just "extends MyParent"
3. Write some specifications in that class
4. Attempt to run this MyChild as part of a group of specifications with
the SpecsFinder

What is the expected output? What do you see instead?

I would expect MyChild to be run along with all my other specs. It is
skipped silently. (However, if you run MyChild in Intellij, it works fine)


What version of the product are you using? On what operating system?

1.5.0 on OSX, but I'm pretty sure this is happening on Windows as well.

Please provide any additional information below.


```

Original issue reported on code.google.com by `michaelnash01` on 6 Aug 2009 at 6:30

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r1076.
```

Original comment by `etorrebo...@gmail.com` on 6 Aug 2009 at 10:57
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Please test version 1.6.0-SNAPSHOT and tell me how it works for you.
```

Original comment by `etorrebo...@gmail.com` on 6 Aug 2009 at 10:58

**Comment by GoogleCodeExporter** [comment]
```
Eric: Yes, fixed nicely in 1.6.0-SNAPSHOT. Thanks!
```

Original comment by `michaelnash01` on 10 Aug 2009 at 2:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 10 Aug 2009 at 2:41
- Changed state: **Verified**
